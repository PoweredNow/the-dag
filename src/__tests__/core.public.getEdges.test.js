const addEdges = require('../core/public/addEdges');
const addNodes = require('../core/public/addNodes');
const getEdges = require('../core/public/getEdges');

// const { getState } = require('../core/private/_state');

describe('core.public.getEdges', () => {
  test('exports', () => {
    expect(getEdges).toMatchSnapshot();
  });
  test('Gets edges', () => {
    addNodes([{ nodeID: 1 }, { nodeID: 2 }]);
    addEdges([{ source: { nodeID: 1 }, target: { nodeID: 2 } }]);
    expect(getEdges()).toMatchSnapshot();
    // expect(state).toMatchSnapshot();
  });
});
