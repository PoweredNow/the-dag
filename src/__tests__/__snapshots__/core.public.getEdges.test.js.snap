// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`core.public.getEdges Gets edges 1`] = `
Object {
  "1_2": Object {
    "edgeData": Object {},
    "edgeID": "1_2",
    "sourceID": 1,
    "targetID": 2,
  },
}
`;

exports[`core.public.getEdges exports 1`] = `[Function]`;
